src/react-components/profile-entry-panel.js

```diff
@@ -189,32 +143,32 @@ class ProfileEntryPanel extends Component {
               />
             )}
 
-            <div className={styles.tabs}>
-              <a
-                onClick={() => this.setState({ avatarType: "legacy", avatarId: "botdefault" })}
-                className={classNames({ selected: this.state.avatarType === "legacy" })}
-              >
-                <FormattedMessage id="profile.tabs.legacy" />
-              </a>
-              <a
-                onClick={() => this.setState({ avatarType: "skinnable", avatarId: null })}
-                className={classNames({ selected: this.state.avatarType === "skinnable" })}
-              >
-                <FormattedMessage id="profile.tabs.skinnable" />
-              </a>
-              <a
-                onClick={() => this.setState({ avatarType: "url", avatarId: "" })}
-                className={classNames({ selected: this.state.avatarType === "url" })}
-              >
-                <FormattedMessage id="profile.tabs.url" />
-              </a>
+            <a
+              className={styles.chooseAvatar}
+              onClick={() => this.props.mediaSearchStore.sourceNavigateWithNoNav("avatars")}
+            >
+              <FormattedMessage id="profile.choose_avatar" />
+            </a>
+
+            <div className={styles.preview}>
+              <AvatarPreview avatarGltfUrl={this.state.avatarGltfUrl} />
+
+              {this.state.avatarType === AVATAR_TYPES.SKINNABLE && (
```

Comment:
We will need to remember to update this when we add the owner id to the avatar response so that "real" featured avatars dont get an edit button.